server/tests/api/test_jurisdictions.py

```diff
@@ -512,6 +512,8 @@ def test_last_login_by_jurisdiction_most_recent(client: FlaskClient, election_id
         "a2@example.com",
     ]
 
+    db_session.expunge(election)
```

Comment:
Without this `expunge` we were getting this error for the `Election` model:

https://docs.sqlalchemy.org/en/13/errors.html#error-bhk3